github/gists.go

```diff
@@ -30,6 +30,7 @@ type Gist struct {
 	GitPushURL  *string                   `json:"git_push_url,omitempty"`
 	CreatedAt   *time.Time                `json:"created_at,omitempty"`
 	UpdatedAt   *time.Time                `json:"updated_at,omitempty"`
+	NodeID	    *string		  `json:"node_id,omitempty"`
```

Comment:
This line and many others aren't gofmted. You'll want to use [`gofmt`](https://golang.org/cmd/gofmt/) command to format all these files. You can either invoke it directly in this directory:

```
gofmt -l -w .
```

Or you can format the entire package via `go`'s `fmt` subcommand:

https://golang.org/cmd/go/#hdr-Run_gofmt_on_package_sources

```
go fmt github.com/google/go-github/github
```

Reply:
Thanks @shurcooL  Resolved!